tools/test-case-generators/genall.go

```diff
@@ -0,0 +1,515 @@
+// Standalone executable for generating all test manifests in parallel.
```

Comment:
Something for a follow-up: I personally don't like having random Go files around, let's move this to `/cmd`? I'm also in favor of splitting the worker queue to a separate file (not package, just a file) to keep these files smaller.